app/v1/models/request/retrieveSelfAssessmentTransaction/RetrieveSelfAssessmentTransactionsRawRequest.scala

```diff
@@ -0,0 +1,6 @@
+
+package v1.models.request.retrieveSelfAssessmentTransaction
+
+import v1.models.request.RawData
+
+case class RetrieveSelfAssessmentTransactionsRawRequest(nino: String, from: String, to: String) extends RawData
```

Comment:
Please make `from` and `to` optional so they can be validated as "there" or "not there" in the request parser/validator.

Reply:
(only in the raw model though... they should stay as mandatory in the parsed request model)